src/guides/v2.4/graphql/mutations/delete-requisition-list.md

```diff
@@ -0,0 +1,74 @@
+---
+group: graphql
+title: deleteRequisitionList mutation
+b2b_only: true
+---
+The `deleteRequisitionList` mutation deletes a requisition list of the logged in customer.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
```

Comment:
I don't think we need this type of statement for mutations that update/delete an object. The fact that you're supplying an ID of an existing object says the feature is supported.